src/systempolicy/systemPolicy.go

```diff
@@ -985,6 +986,8 @@ func updateSysPolicies() {
 
 	wpfsPolicies := populateKnoxSysPolicyFromWPFSDb("", "", "", "")
 
+	go insertSysPoliciesYamlToDB(wpfsPolicies)
```

Comment:
Shouldn't this API be called below after checking for duplicate policies?
Secondly, why the use of go routine?
In fact it would be better to call this API from inside `libs.InsertSystemPolicies()`.

Reply:
@nyrahul It is not possible to call the insertSysPoliciesYamlToDB() inside libs.InsertSystemPolicies() as this will cause circular import error (importing libs in systempolicy and systempolicy in libs). 

This API is called inside updateSysPolicy() and I hope it to not have any duplicate policies.